.github/workflows/main.yml

```diff
@@ -40,10 +40,7 @@ jobs:
           python -m pip install dist/*.whl
 
       - name: Run tests
-        run: |
-          mkdir testdir
-          cd testdir
-          python -m pytest -s -v --cov=httpstan --cov-fail-under=93 ../tests
+        run: python -m pytest -s -v --cov=httpstan tests
```

Comment:
I guess this is ok. I think running from different dir than the main dir, helps in catching the  relative path / imports issues if there are any.